app/assets/stylesheets/scaffolds.scss

```diff
@@ -98,7 +98,7 @@ body.mod-status-true .mod-icon-visible-block {
 }
 
 body.trusted-status-true .trusted-visible-block {
-  display: flex !important;
+  display: block !important;
```

Comment:
I don't _think_ this causes any issues, but in the other place that this is used (the nav menu dropdown) the other links use flex layout?

Reply:
Oh yeah, I meant to comment on that: Yes @filleduchaos this does not cause any issues. The only other place it uses it is indifferent to block vs flex.